### PR TITLE
Add commands v4-init, v4-dump

### PR DESCRIPTION
**Pull request by shayne-fletcher**
- admin-cli
  - add `v4-init`, `v4-dump` commands
- mv test-setup.json v3-test-setup.json
- add v4-test-setup.json

### CODE REVIEW

File: admin-cli/src/index.ts [hunk 1]
```diff
@@ -315,14 +318,87 @@ namespace v4 {
...
+  }
+
 }//namespace v4
 
 // eslint-disable-next-line @typescript-eslint/no-namespace
 namespace cli {
 
   type Cli =
     | { command: 'v3-init'; file: string }
+    | { command: 'v4-init'; file: string }
```
Comment (hurryabit) [inline admin-cli/src/index.ts]: That's a duplicate of the last command.

File: admin-cli/src/index.ts [hunk 2]
```diff
@@ -403,6 +483,16 @@ namespace cli {
       })
       .example('$0 v3-init -f test-setup.json',
                'Ledger initialization from configuration file.')
+      .command({
```
Comment (hurryabit) [inline admin-cli/src/index.ts]: That's also a duplicate.

File: admin-cli/src/index.ts [hunk 3]
```diff
@@ -259,6 +259,7 @@ namespace v3v4 {
 
 // eslint-disable-next-line @typescript-eslint/no-namespace
 namespace v4 {
+
```
Comment (hurryabit) [inline admin-cli/src/index.ts]: 😿

Comment (shayne-fletcher) [reply admin-cli/src/index.ts]: why the long face? newlines?

Comment (shayne-fletcher) [reply admin-cli/src/index.ts]: diff noise. apologies.

**Pull request merged**